src/Microsoft.AspNetCore.AzureAppServicesIntegration/AppServicesWebHostBuilderExtensions.cs

```diff
@@ -19,9 +20,20 @@ public static IWebHostBuilder UseAzureAppServices(this IWebHostBuilder hostBuild
             {
                 throw new ArgumentNullException(nameof(hostBuilder));
             }
-#pragma warning disable 618
-            hostBuilder.ConfigureLogging(builder => builder.AddAzureWebAppDiagnostics());
-#pragma warning restore 618
+            hostBuilder.ConfigureLogging(builder => builder
+                .AddAzureWebAppDiagnostics()
+                .AddEventSourceLogger()
+            );
+            hostBuilder.ConfigureServices(collection => collection.Configure<LoggerFilterOptions>(
+                options =>
+                {
+                    // Enable event source logger provide for all categories at Information level
```

Comment:
Only Information? In theory, the EventSource logger has it's own filtering mechanic so maybe we should just pump all messages through to it.

Reply:
Yes, I was playing with this last night. I don't know why we need this. Can't this be controlled from the outside? How does the end to end work here?

Reply:
https://github.com/aspnet/Logging/issues/877

Filters are applied before messages get to the event logger. We decided to let only info through for 2.2 and fix end to end for 3.0

Reply:
Yeah, we need to revisit the EventSourceLogger in 3.0 for a few reasons if we're going to make it part of an end-to-end diagnostics egress story.